content/en/content-management/mathematics.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Mathematics in markdown
+linkTitle: Mathematics
+description: Include mathematical equations and expressions in your markdown using LaTeX or TeX typsetting syntax.
+categories: [content management]
+keywords: [chemistry,latex,math,mathjax,tex,typsetting]
+menu:
+  docs:
+    parent: content-management
+    weight: 250
+weight: 250
+toc: true
+math: true
+---
+
+$$
+KL(\hat{y} || y) = \sum_{c=1}^{M}\hat{y}_c \log{\frac{\hat{y}_c}{y_c}}
```

Comment:
```suggestion
KL(\hat{y} || y) = \sum_{c=1}^{M}\hat{y}_c \log{\frac{\hat{y}_c}{y_c}} \\
```

Adding `\\` to the end will render as a line break, which is I think what you want. Even better would be to use the following for a more precise alignment of the equals signs:

```
$$
\begin{aligned}
KL(\hat{y} || y) &= \sum_{c=1}^{M}\hat{y}_c \log{\frac{\hat{y}_c}{y_c}} \\ 
JS(\hat{y} || y) &= \frac{1}{2}(KL(y||\frac{y+\hat{y}}{2}) + KL(\hat{y}||\frac{y+\hat{y}}{2}))
\end{aligned}
$$
```

$$
\begin{aligned}
KL(\hat{y} || y) &= \sum_{c=1}^{M}\hat{y}_c \log{\frac{\hat{y}_c}{y_c}} \\ 
JS(\hat{y} || y) &= \frac{1}{2}(KL(y||\frac{y+\hat{y}}{2}) + KL(\hat{y}||\frac{y+\hat{y}}{2}))
\end{aligned}
$$

Ditto for the preformatted equation in lines 28-31